models/hospital.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {
+    static associate(models) {
+      Hospital.hasMany(models.HospitalAdministrator);
```

Comment:
@francisli I was reading here: https://sequelize.org/master/manual/assocs.html that Sequelize recommends defining associations in pairs. That way we could use `hospital.getHospitalAdmins()`. I'm assuming queries from the Hospital to the HospitalAdmin are a lot more expensive than the reverse, but I don't think that will be an issue at our size. Does this seem like a good add?

Reply:
Yes, do add the reverse associations...!